src/ResourceManager/Dns/Commands.Dns.Test/ScenarioTests/ZoneTests.cs

```diff
@@ -31,36 +31,31 @@ public void TestZoneCrud()
             DnsTestsBase.NewInstance.RunPowerShellTest("Test-ZoneCrud");
         }
 
-        [Fact]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Fact(Skip = "Test is failing, service team needs to re-record")]
```

Comment:
@Nilambari why are these tests failing? @muwaqar would you mind working with Nilambari to enable these tests?

Reply:
Enabled skipped tests back as tests are passing now.